src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2825,4 +2825,28 @@ We use App Center to track which settings are being used, find bugs, and fix cra
   <data name="BehaviorForOpeningFilesAndFolders" xml:space="preserve">
     <value>Behavior for opening files and folders</value>
   </data>
+  <data name="RestoreRecycleBin" xml:space="preserve">
+    <value>Restore Recycle Bin</value>
+  </data>
+  <data name="ConfirmRestoreBinDialogContent" xml:space="preserve">
+    <value>Are you sure you want to restore all items in the recycle bin?</value>
```

Comment:
I don't think we need confirmation for restoring items right now but happy to discuss this.

Reply:
May be overkill, but may prevent some unwanted action. Clicking on restore instead of emptying, and scatter back all the items we wanted to remove back on the computer.

Reply:
Let's keep it 👍